src/workflow/summary_merge_request.py

```
@@ -54,17 +58,61 @@ async def prep_async(self, shared):
+            raw_diff = get_merge_request_raw_diff(project_id, merge_number)
+            shared["raw_diff"] = raw_diff
+            # 获取所有 commits
+            commits = get_merge_request_commits(project_id, merge_number)
+            start_commit_hash = commits[-1]["short_id"]
+            end_commit_hash = commits[0]["short_id"]
+            shared["start_commit_hash"] = start_commit_hash
+            shared["end_commit_hash"] = end_commit_hash
+            shared["commits"] = "\n".join(
+                [f"{commit['short_id']}\n{commit['message']}\n" for commit in commits]
+            )
+            return shared
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)